examples/3rd.js

```diff
@@ -0,0 +1,44 @@
+/*
+slide html will execute this js file.
+Here to load plugins js file and other needed assets,
+then set revealConfig,
+after this executed, slide html will call
+window.addEventListener("load", function(){Reveal.initialize(revealConfig);}); 
+*/
+
+//Chalkboard
+new_element = document.createElement("link");
+new_element.rel = "stylesheet";
+new_element.href = "https://use.fontawesome.com/releases/v5.15.3/css/all.css";
+document.getElementsByTagName("head")[0].appendChild(new_element);
+
+new_element = document.createElement("link");
+new_element.rel = "stylesheet";
+new_element.href = "./reveal.js/plugin/chalkboard/style.css";
+document.getElementsByTagName("head")[0].appendChild(new_element);
+
+new_element = document.createElement("script");
+new_element.type = "text/javascript";
+new_element.src = "./reveal.js/plugin/chalkboard/plugin.js";
+new_element.onload = function () {
+  revealConfig.plugins.push(RevealChalkboard);
+  revealConfig.chalkboard={ // font-awesome.min.css must be available
+    src: "chalkboard/chalkboard.json",
+    storage: "chalkboard-demo",
+    toggleChalkboardButton: { left: "80px" },
+    toggleNotesButton: { left: "130px" },
+    colorButtons: 5
+  }
+
+};
+document.getElementsByTagName("head")[0].appendChild(new_element);
```

Comment:
I don't think this is the right way to do it, we should use docinfo file to load additional plugins. We don't need to add script and style with JavaScript, we can declare them as HTML:

**docinfo.html**
```html
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.3/css/all.css"/>
<link rel="stylesheet" href="./reveal.js/plugin/chalkboard/style.css"/>
<script src="./reveal.js/plugin/chalkboard/plugin.js">
<script>
  // ...
  Reveal.configure()
</script>
```

Reply:
Yes it is clearer when using docinfo. I have updated with docinfo-footer.html because reveal.js initialization takes place at the end of body tag, I think it is safer to keep the paradigm.